hibernate-core/src/main/java/org/hibernate/Filter.java

```diff
@@ -95,4 +95,12 @@ public interface Filter {
 	 * @return The flag value
 	 */
 	boolean isAutoEnabled();
+
+	/**
+	 * Get the associated {@link FilterDefinition applyToLoadByKey} of this
+	 * named filter.
+	 *
+	 * @return The flag value
+	 */
+	boolean isApplyToLoadByKey();
```

Comment:
`applyToLoadByKey` or `isAppliedToLocadByKey`

Reply:
I agree that `applyToLoadByKey` is "better", but marginally so.  I'm +/-0 either way

Reply:
The problem is that "is apply" isn't grammatically correct.

Reply:
Tell that to JavaBeans